pkg/volume/cephfs/cephfs.go

```diff
@@ -288,12 +289,15 @@ func (cephfsVolume *cephfs) GetPath() string {
 	return cephfsVolume.plugin.host.GetPodVolumeDir(cephfsVolume.podUID, utilstrings.EscapeQualifiedNameForDisk(name), cephfsVolume.volName)
 }
 
-// GetKeyringPath creates cephfuse keyring path
-func (cephfsVolume *cephfs) GetKeyringPath() string {
+// getKeyringPath creates cephfuse keyring file name
+func (cephfsVolume *cephfs) getKeyringFileName() string {
+	return cephfsVolume.id + ".keyring"
```

Comment:
IIUC, this `id` is not unique, perhaps it's better to use some unique string? cc @rootfs

Reply:
The keyring file will be deleted after mount, whenever it succeed or failed.